docs/content/ecs/logging.md

```diff
@@ -13,7 +13,7 @@ All workers use the `ForwardingDispatcher` by default. To replace it with the `L
 
 ### Using the dispatcher
```

Comment:
Maybe change "dispatcher" to "ILogDispatcher"? My initial reaction when reading this was "what is "the dispatcher" referring to?"